### PR TITLE
ramips: add TP-Link TL-WR802N-v4 support

### PR DESCRIPTION
**Pull request by majo-icutech**
This patch adds support for the TP-Link TL-WR802N-v4.
https://wiki.openwrt.org/toh/tp-link/tl-wr802n

### CODE REVIEW

File: target/linux/ramips/base-files/etc/board.d/02_network [hunk 1]
```diff
@@ -152,6 +152,7 @@ ramips_setup_interfaces()
 	mzk-ex750np|\
 	na930|\
 	pbr-d1|\
+  tl-wr802n-v4|\
```
Comment (ynezz) [inline target/linux/ramips/base-files/etc/board.d/02_network]: @majo-icutech Whitespace issue.

**Comment by ynezz** [comment]
@majo-icutech Please take a look at https://openwrt.org/submitting-patches and fix your commit message, it's missing following

> If you add support for new hardware: Include in your commit message a short description of the hardware and how to install OpenWrt on it. Have a look at the [recent additions](https://git.openwrt.org/?p=openwrt/openwrt.git&a=search&h=HEAD&st=commit&s=add+support+for) for some examples.

File: target/linux/ramips/dts/TL-WR802NV4.dts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/dts-v1/;
```
Comment (ynezz) [inline target/linux/ramips/dts/TL-WR802NV4.dts]: @majo-icutech Please can you add here [license specifier](https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/BDCOM-WAP2100-SK.dts#L1)? For some example just do `git grep SPDX target/linux/ramips/dts`

**Comment by majo-icutech** [comment]
How can I reopen the PR?

**Comment by ynezz** [comment]
Scroll down to the bottom of the pull request and there should be `Reopen pull request` button or something like that.

**Comment by majo-icutech** [comment]
grafik

Any idea?

**Comment by ynezz** [comment]
Ah, so you can't do that now, you indeed need to create a new PR now.

**Pull request closed without merging**